### DISCUSSION TITLE
Ender-3 V2 Neo

### DISCUSSION THREAD

**Discussion by DoogH2004** [Jyers/Marlin#2002] [Q&A]
Hello!

I have the Ender-3 V2 Neo.  I am not sure what configurations I need to set in the config files to support this device.  To be clear, the device supports CR Touch.

Has anyone played with this?

Thanks!

**Comment by Ergonomicmike** [comment]
I can't tell if you're a newbie like I am. I'll assume "yes," since you would already know what I'm going to tell you if you weren't like me.

You will have to open up the cover to see the motherboard, to see what version board it is. From what I, as a newbie know, the 4.2.7 boards are the best, but they don't ship with them anymore. You have to buy them as an upgrade. 

The 4.4.2 boards can be okay. That's what's in mine. They're "quiet steppers."

I just saw someone here in the Discussion board show a photo of his Ender, and it has 4.3.1. (Or something like that.)  No idea. 

Although it seems that, as time goes by, Creality is "cheaping out" on components, either to save money or because of the 'chip shortage.' (Or both.) For example, I just discovered the other day that my Display board doesn't have as much memory as the original Display boards had.

In any event, the code bases here are written for certain boards. So you need to know the Version number of your motherboard in order to find a matching code base.
